scripts/requirements.txt

```diff
@@ -1,3 +1,3 @@
-mkdocs==1.1.2
+mkdocs==1.0.4
```

Comment:
Don't versions count up?

Reply:
1.1.2 causes problems, apparently its not fully compatible with the way we make the mkdocs.yaml file.  Once I get a chance to look at why I will fix it and move it up.